static/faq.html

```diff
@@ -291,6 +297,35 @@ <h2 id="default-connections">
                 </li>
             </ul>
 
+            <h2 id="verified-boot">
+                <a href="#verified-boot">Verified Boot</a>
+            </h2>
+
+            <h3 id="orange-warning-screen">
+                <a href="#orange-warning-screen">My phone displays an orange "software
+                        integrity not guaranteed" warning. What's wrong?</a>
+            </h3>
+
+            <p>If you see this orange screen, the phone is telling you that the boot loader has been
+            left unlocked, which disables security measures such as Android Verified Boot. This
+            allows persistent changes to be written to the system image at will. During the
+            installation process of GrapheneOS, unlocking the bootloader is an important step to
+            allow the installer to run, but it is crucially important that as soon as the install
+            script has completed and the new operating system boots, to lock the bootloader again.
+            Since all installs of GrapheneOS are end-to-end signed, verified, and bitwise identical,
+            locking the bootloader again is considered to be a part of the installation. If you see
+            this screen, your installation is not yet completed.</p>
+
+            <p>Visit the
+            <a href="https://grapheneos.org/install#locking-the-bootloader">installation guide on
+            locking the bootloader</a> to find out how to fix this issue. Each time the bootloader
+            is locked or unlocked, as a safeguard the phone will be reset and all the data stored on
+            the phone, such as photos, E-mails, text messages, saved passwords, and your Two-Factor
```

Comment:
```suggestion
            the phone, such as photos, e-mails, text messages, saved passwords, and your two-factor
```

Reply:
Huh, odd, I always thought it was capitalized...?

Reply:
Actually, I think most places use "email" now instead of "e-mail". Gmail and Outlook use "email" on their front pages.